go/Makefile

```diff
@@ -100,6 +15,20 @@ test:
 	/usr/local/pf/t/pfconfig-test ;\
 	PFCONFIG_TESTING=y go test -count=1 ./...
 
+CMDS = $(wildcard cmd/*)
+BINARYS = $(patsubst cmd/%, %, $(CMDS))
+.PHONY: $(BINARYS)
+$(BINARYS):
+	cd cmd/$@ && \
+	go build && \
+	mv $@ ../..
+
+BINARYS_RACE = $(patsubst cmd/%, %-race, $(CMDS))
+$(BINARYS_RACE):
+	cd cmd/$(@:-race=) && \
+	go build -o $@ -race && \
+	mv $@ ../..
+
```

Comment:
`GOBINARIES` will need to be replace by `BINARYS` in this Makefile and also in `config.mk` in order to build `pfcron`. If we do this, we can remove `BINARYS` from `config.mk` because it doesn't seems to be used anywhere.

Reply:
pfcron is not merged into devel yet.

Reply:
GOBINARIES is for go binaries that will be placed in sbin.
BINARIES is for all the binaries including db-tester and radius-tester.
Should we rename it to GOBINARIES GOSBINBINARIES?

Reply:
Got your point. In that case, I suggest to:
- replace BINARIES by ALL_BINARIES
- replace GOBINARIES by PF_BINARIES

to be more explicit.